src/OperatingSystemApp/controlpanel.js

```diff
@@ -0,0 +1,13 @@
+/* exported OSControlPanel */
+
+const {GObject, Gtk} = imports.gi;
+
+var OSControlPanel = GObject.registerClass({
+    GTypeName: 'OSControlPanel',
+    Template: 'resource:///com/endlessm/HackToolbox/OperatingSystemApp/controlpanel.ui',
+}, class OSControlPanel extends Gtk.Box {
+    _init(props = {}) {
```

Comment:
Does not look like it's required to override `_init()` here.

Reply:
no, not right now